### ISSUE TITLE
Request Line is too large

### ISSUE THREAD

**Issue by aray** [apache/airflow#8636]
With 100 dags per page and dag names that average over 40 characters we are running into the following bug on the four endpoints below that are used by the UI.

/blocked
/last_dagruns
/dag_stats
/task_stats

Request URL is like:
https://myhostname/last_dagruns?dag_ids=massive,long,list,of,dags,that,is,over,4094,chars

and response is status 400 (bad request) with body:

```
<html>
  <head>
    <title>Bad Request</title>
  </head>
  <body>
    <h1><p>Bad Request</p></h1>
    Request Line is too large (4385 &gt; 4094)
  </body>
</html>
```

I believe we can change the gunicorn setting to fix this, see: https://stackoverflow.com/questions/40625360/how-to-set-gunicorn-limit-request-line-parameter-over-8190

This is observed in v 1.10.9

I believe this was introduced in https://issues.apache.org/jira/browse/AIRFLOW-6095 (in 1.10.7) and other similar changes for the other endpoints that added the query parameter.

For now we worked around this issue by changing our page length to 50 dags.

**Comment by ashb** [comment]
Fixed by #7364 which was released in 1.10.10.

**Comment by aflag** [comment]
I don't think this actually made it to 1.10.10, but only to airflow 2
